core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -205,6 +206,11 @@ angular.module('oppia').directive('tutorCard', [
             $scope.waitingForOppiaFeedback = false;
             $scope.windowDimensionsService = WindowDimensionsService;
             $scope.isIframed = UrlService.isIframed();
+            $scope.audioBarIsExpandedAndOnMobileDevice = function() {
+              return (
+                AudioBarStatusService.isAudioBarExpanded() &&
+                DeviceInfoService.isMobileDevice());
```

Comment:
Check ` DeviceInfoService.isMobileDevice()` first as that's the main requiremnt here, right?